### PR TITLE
note with link to session certificates

### PR DESCRIPTION
**Pull request by nadinequinr3**

### CODE REVIEW

**Review by yarivus — APPROVED** [review]
LGTM

File: content/en/platform/corda/5.0-beta/deploying/deployment-tutorials/onboarding/mgm-onboarding.md [hunk 1]
```diff
@@ -231,6 +231,10 @@ If using Bash, the result contains `session key ID` (e.g. 3B9A266F96E2). Run the
 ```shell
 export SESSION_KEY_ID=<session key ID>
 ```
+{{< note >}}
+You can use a certifcate in addition to the session initiation key pair. For more information, see [Configuring Optional Session Certificates](session-certificates.html).
```
Comment (paulmoloneyr3) [inline content/en/platform/corda/5.0-beta/deploying/deployment-tutorials/onboarding/mgm-onboarding.md:235]:
```suggestion
You can use a certificate in addition to the session initiation key pair. For more information, see [Configuring Optional Session Certificates](session-certificates.html).
```

**Review by paulmoloneyr3 — CHANGES_REQUESTED** [review]

File: content/en/platform/corda/5.0-beta/deploying/deployment-tutorials/onboarding/mgm-onboarding.md (hunk 1 above)
Comment (nadinequinr3) [reply content/en/platform/corda/5.0-beta/deploying/deployment-tutorials/onboarding/mgm-onboarding.md]: Doh! Thanks @paulmoloneyr3

**Review by yarivus — APPROVED** [review]
LGTM

**Review by paulmoloneyr3 — APPROVED** [review]

**Review by yarivus — COMMENTED** [review]
LGTM - clean merge

**Pull request merged**